frontend/.husky/pre-commit

```diff
@@ -3,7 +3,7 @@
 
 echo '🏗️👷 Linting, testing and building your project before committing'
 
-cd frontend && yarn lint-staged
+# cd frontend && yarn lint-staged
```

Comment:
why?

Reply:
Was blocking my push and I was a little impatient. but I've uncommented it. I'll run linter on my code to make sure the pre-commit works